crates/term_input/src/lib.rs

```diff
@@ -110,16 +111,29 @@ byte_seq_parser! {
     [8] => Key::Backspace,
     [127] => Key::Backspace,
     [1] => Key::Ctrl('a'),
+    [2] => Key::Ctrl('b'),
```

Comment:
Added more supported keys. Did not verify them all, but a few I did verify with `showkey -a`. 
If more keys/combos are useful I can add them too